.github/ISSUE_TEMPLATE/lead.md

```diff
@@ -24,7 +24,7 @@ hold and only released once approved by a SIG Release (subproject) owner.**
   - [ ] Has joined the following mailing lists:
     - [kubernetes-sig-release](https://groups.google.com/forum/#!forum/kubernetes-sig-release)
     - [kubernetes-release-team](https://groups.google.com/a/kubernetes.io/g/release-team)
-    - [kubernetes-dev](https://groups.google.com/forum/#!forum/kubernetes-dev)
+    - [kubernetes-dev](https://groups.google.com/a/kubernetes.io/group/dev)
```

Comment:
nit:
```suggestion
    - [kubernetes-dev](https://groups.google.com/a/kubernetes.io/g/dev)
```

Reply:
@saschagrunert I did a find and replace on the repo to replace the url.
The url (`…/goup/dev`) that I used is the one that is in the old mailing list telling users with a message and this url.

It looks like both are correct `…/group/dev` and `…/g/dev` the first one redirects to the second one with the `g`.